rust-runtime/aws-smithy-types/src/error.rs

```diff
@@ -12,7 +12,7 @@ pub mod interceptor;
 pub mod metadata;
 mod unhandled;
 
-pub use interceptor::InterceptorError;
+pub use interceptor::Error;
```

Comment:
Prefer it be called `InterceptorError`. With it in `crate::error`, there's no way to tell it has anything to do with interceptors... should it also be in the `aws-smithy-interceptor-api` crate?

Reply:
Ok, I'll move it.